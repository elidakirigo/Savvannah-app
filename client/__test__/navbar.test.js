import Navbar from '../components/navbar'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom'

jest.mock('next-auth/react')

describe('Navbar render', () => {
	it('renders correctly', () => {
		const { container } = render(<Navbar />)
		expect(container).toMatchSnapshot()
	})
})
