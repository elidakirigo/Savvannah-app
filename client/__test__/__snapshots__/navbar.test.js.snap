// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar render renders correctly 1`] = `
<div>
  <div
    class=" w-full bg-purple-600 text-white p-3  flex items-center justify-start"
  >
    <a
      class="flex flex-row-reverse gap-4 items-center justify-center"
      href="/"
    >
      <h1>
         
      </h1>
      <img
        alt="user"
        class="rounded-full ml-1"
        data-nimg="1"
        decoding="async"
        height="30"
        src=""
        style="color: transparent;"
        width="30"
      />
    </a>
    <div>
      <a
        class=" text-white p-3 mx-auto"
        data-testid="home"
        href="/home"
      >
        Home
      </a>
      <button>
        signOut
      </button>
    </div>
  </div>
</div>
`;
